src/components/PatientLine.tsx

```diff
@@ -1,49 +1,106 @@
-      className={isValidIconClass}
-    />
-  )
+  const isValidIconClass = prescriptionIsValid ? 'success' : 'danger'
+  const [show, setShow] = useState(false)
+  const { addError } = useContext(ErrorContext)
+  const handleClose = (): void => setShow(false)
+  const handleShow = (): void => setShow(true)
+
+  const goToPatient = (): void => {
+    navigate(`/patients/${patient.id}`)
+    handleCloseButtonsModal()
+  }
+  const handleEdit = async (): Promise<void> => {
+    navigate(`/patients/edit/${patient.id}`)
+    handleCloseButtonsModal()
+  }
 
-  const goToPatient = (id: number): void => {
-    navigate(`/patients/${id}`)
+  const onDelete = async (): Promise<void> => {
+    assert(token)
+    try {
+      await deletePatient(token, patient.id)
+      navigate('/patients')
+    } catch (error) {
+      console.error(error)
+      addError({ body: 'Error deleting patient' })
+    }
+    handleClose()
   }
 
   return (
-    <Card onClick={() => goToPatient(patient.id)} style={{ cursor: 'pointer' }} className='mt-2 shadow'>
-      <Card.Body>
-        <Card.Title>{patient.lastname}</Card.Title>
-        <Card.Subtitle className='mb-2 text-muted'><small>{patient.address} à {patient.city}</small></Card.Subtitle>
-        <Card.Text>
-          <>Renouvellement le: {prescriptionEndDate} - {icon}</>
-          {patient.prescriptions.map((prescription) => (
-            <small key={prescription.id}>
-              Dr. {prescription.prescribing_doctor}
-            </small>
-          ))}
-        </Card.Text>
-      </Card.Body>
-    </Card>
+    <>
+      <Card className='shadow mt-4'>
+        <Card.Body>
+          <div className='row align-items-start'>
+            <div className='col'>
+              <div className='d-flex align-items-center'>
+                <Badge bg='secondary' text='dark' className='me-2'>
+                  {patient.lastname} {patient.firstname}
+                </Badge>
+                <div className='d-flex ms-auto align-items-center'>
+                  <FontAwesomeIcon
+                    icon={['fas', 'ellipsis']}
+                    onClick={handleShowButtonsModal}
+                    style={{ cursor: 'pointer' }}
+                  />
+                </div>
+              </div>
+              <hr className='my-2' />
+              <div className='d-flex align-items-center mb-3'>
+                <small><strong>{t('text.endPrescription')}</strong> <Badge pill bg={isValidIconClass}> {prescriptionEndDate}</Badge></small>
+              </div>
+            </div>
+          </div>
+        </Card.Body>
+      </Card>
+
+      <Modal show={buttonsModalShow} onHide={handleCloseButtonsModal}>
+        <Modal.Header closeButton>
+          <Modal.Title>{t('text.actionsPatients')}</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Button size='sm' variant='dark' className='me-2' onClick={goToPatient}>
+            {t('text.seePatient')}
+          </Button>
```

Comment:
To me this shouldn't be part of this popup, but accessible via a simple click on the card

Reply:
oki I going to change